### DISCUSSION TITLE
Allow users to configure the life of Shortened Links

### DISCUSSION THREAD

**Discussion by danstadler-pdx** [grafana/grafana#62735] [Explore]
As of today, Shortened Links have a fixed lifespan of 7 days, per:
https://grafana.com/docs/grafana/latest/explore/#share-shortened-link

For some use cases, this is unhelpful - specifically when wanting to use these links in areas like runbooks, where the link might not get used as often as weekly, but is critical to have ready when an issue arises.

Perhaps the TTL on these links could be configurable; alternatively there could be some way to pair Grafana with an external link shortener, in order to take advantage of existing solutions as much as possible.

**Comment by Elfo404** [comment]
Closing this as duplicate of https://github.com/grafana/grafana/discussions/37331
